src/utils/jwtUtils.ts

```diff
@@ -0,0 +1,17 @@
+import jwt from 'jsonwebtoken'
+import { JWT_SECRET, JWT_EXPIRE_TIME } from '../config/index'
+
+export const generateToken = (data: object) => {
+  const token = jwt.sign(data, JWT_SECRET, {
+    expiresIn: JWT_EXPIRE_TIME,
+  })
+  return token
+}
+
+interface JwtPayload {
+  userId: string
+  id: string
+}
```

Comment:
I think this file was created before by Eric, Why did you add this part?

Reply:
I created a datatype for JWT token decoded content